docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -218,7 +222,7 @@ class JsonStreamingExamplesSpec extends RoutingSpec with CompileOnlySpec {
               .runFold(0) { (cnt, _) => cnt + 1 }
 
           complete {
-            measurementsSubmitted.map(n => Map("msg" -> s"""Total metrics received: $n"""))
+            measurementsSubmitted.map(n => s"""Total metrics received: $n""")
```

Comment:
Another suspicious change

Reply:
This change was part of 63b3437048. It might indicate a change in behavior, but likely it's a change in behavior between Scala versions rather than between before and after the changes to the module. Since the `Map` to json encoding doesn't seem super relevant to this particular test it seemed more productive to just simplify it to a regular string.

Reply:
Right, so if I understand correctly there probably was some issue with construction of `Map` between Scala 2 and Scala 3 and the solution was "screw it, lets just not use a `Map` because its not important and its just in tests anyways.

Reply:
Yes - specifically in having the implicits correctly pick up that the `Map` can automatically be converted to a Json object and be Marshallable. It's not useless to figure out and show how to make that work again, but the risk that this is a bug that needs to be fixed rather than just something that would need a change in the test code to work on Scala 3 feels relatively low.

Reply:
I wouldn't be surprised if this was since solved in later Scala versions, there is some merit in spending a low amount of time to figure out if its solved but I will look into this when a future PR updating pekko to Scala 3.3 LTS is out.

Thanks for spending time looking into it!

Reply:
I checked that "just" switching back to a map is not sufficient even on 3.3.0. Might be just a matter of bringing into scope the right implicit, but I don't think I'll dive into that now.